crates/ef-testing/src/models/result.rs

```diff
@@ -18,7 +18,10 @@ pub(crate) fn log_execution_result(
                 warn!("{} reverted:\n{}", case, err.replace("\\n", "\n"));
             } else {
                 info!("{} passed: {:?}", case, info.actual_resources);
+                #[cfg(feature = "v0")]
```

Comment:
we're supposed to have this in kkt1 as well though

Reply:
we do but currently the events implementations don't match, should I leave this and merge as is and wait for v1 update or do you want to wait?